pkg/apis/autoscaling/v1alpha1/pa_defaults_test.go

```diff
@@ -20,6 +20,8 @@ import (
 	"testing"
```

Comment:
Do we need more test-cases here, regarding defaulting the annotations in the HPA case?